pkg/proxy/proxy.go

```diff
@@ -290,15 +312,17 @@ func (r *Redirect) getSourceInfo(req *http.Request) (accesslog.EndpointInfo, acc
 	if !r.l4.Ingress {
 		r.localEndpointInfo(&info)
 	} else if err == nil {
-		parseIPPort(ipstr, &info)
+		if srcIdentity != 0 { //TODO else? parseIPPort?
```

Comment:
Are there situations where this can be srcIdentity == 0? What should be the course of action in that case?

Reply:
`0` is possible when somebody else other than the BPF datapath attempts to connect to the proxy. We should log no source information in that case.